### DISCUSSION TITLE
github education without school email

### DISCUSSION THREAD

**Discussion by samiarcd** [community/community#71548] [GitHub Education]
I am a french student and i applied for the student developer pack. I have given my student id, and my student enrollement lettre signed and dated by our dean. I don't have any school email adress / i just use gmail. I did every step i was asked for but my request keep being rejected. If anyone can help me please

**Comment by hahuyhungdev** [comment]
why you don't have your school email address? hmm

**Reply by samiarcd** [reply]
> why you don't have your school email address? hmm

I unrolled late and they said that it would take time before i get one. And even so later i shouldn't use it unless its for school
